examples/matRad_example15_phantomBuilder.m

```diff
@@ -0,0 +1,152 @@
+%% Example: Generate your own phantom geometry
+%
+% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%
+% Copyright 2023 the matRad development team. 
+% 
+% This file is part of the matRad project. It is subject to the license 
+% terms in the LICENSE file found in the top-level directory of this 
+% distribution and at https://github.com/e0404/matRad/LICENSES.txt. No part 
+% of the matRad project, including this file, may be copied, modified, 
+% propagated, or distributed except according to the terms contained in the 
+% LICENSE file.
+%
+% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+%% In this example we will show
+% (i) how to create arbitrary ct data (resolution, ct numbers)
+% (ii) how to create a cst structure containing the volume of interests of the phantom
+% (iii) generate a treatment plan for this phantom
+
+%% set matRad runtime configuration
+clear all; %somewhat needed for the phantom builder
+matRad_rc; %If this throws an error, run it from the parent directory first to set the paths
+
+%% Create a CT image series
+
+ctDim = [200,200,100]; % y,x,z dimensions
+ctResolution = [2,2,3]; % y,x,z the same here!
+
+%This uses the phantombuilder class, which helps to easily implement simple
```

Comment:
which helps to easily build a water phantom containing geometrical 3D shapes as targets and organs?